templates/project_root_containers/MyProjGenerators/rename.sh

```diff
@@ -59,9 +59,8 @@ find . -name "*.h" -exec sed -e "s/FairStack/${Prefix}Stack/g" $sedstring "{}" "
 sed -e "s#tutorial/PndPythia#$RelativeDir#g" $sedstring CMakeLists.txt
 sed -e "s/PndPythia/$DetectorName/g" $sedstring CMakeLists.txt
 sed -e "s/PndPythia/$DetectorNameUpper/g" $sedstring CMakeLists.txt
-sed -e "s/FAIRROOT_SOURCE_DIR/$ProjectSourceDir/g" $sedstring CMakeLists.txt
 
-if [ -d .svn ]; then  
```

Comment:
Probably can be removed. This comes from a time when we were using Subversion

Reply:
done